recipes/libcurl/all/conanfile.py

```diff
@@ -30,7 +30,9 @@ class LibcurlConan(ConanFile):
                "with_libpsl": [True, False],
                "with_largemaxwritesize": [True, False],
                "with_nghttp2": [True, False],
-               "with_brotli": [True, False]}
+               "with_brotli": [True, False],
+               "enable_bitcode": [True, False]
```

Comment:
I don't think this option belongs to the recipe. it's something that applies to every package, and should be the part of the toolchain IMO. I guess you want either everything to be built with bitcode, or everything without, right?

Reply:
yes, if you want bitcode enabled, you need it for all your dependencies, so you need to set the options for each recipe
how else, if not via options, should this be handled? 
it should generate an extra hash

Reply:
removed, will soon push